technologies/app/vscode/metadata.yaml

```diff
@@ -0,0 +1,25 @@
+version: v1
+type: APP
+id: vscode
+label: VsCode v3.9.0
+baseline: "VsCode v3.9.0"
+description: "VsCode is an IDE that supports several languages (VsCode v3.9.0). \nCreate an environment variable named 'PASSWORD' to set a password."
+available: true
+icon: generic-app
+backgroundColor: "#3EAAF2"
+customFlags: []
+contexts:
+  - id: 1.0
```

Comment:
id must  be unique between all technologies so use something like vscode-3.9.0 for instance